spec/javascripts/models/beta/iteration_spec.js

```diff
@@ -507,286 +831,272 @@ describe("iteration", function() {
+            storyType: "feature"
+          },
+          {
+            id: 9,
+            position: "9",
+            state: "started",
+            estimate: 4,
+            storyType: "feature"
+          },
+          {
+            id: 10,
+            position: "10",
+            state: "started",
+            estimate: 4,
             storyType: "feature"
           },
         ];
 
-        const project = {
-          startDate: moment().format(),
-          iterationLength: 1,
-          defaultVelocity: 8
-        };
-
         const sprints = Iteration.groupBySprints(
           stories,
           project,
-          undefined,
+          initialSprintNumber,
           []
         );
 
-        it('returns 2', () => {
-          expect(sprints[0].points).toBe(2);
+        it("return an array with 1 item", function() {
+          expect(sprints.length).toEqual(1);
         });
 
-        it('returns a number', () => {
-          expect(typeof (sprints[0].points)).toBe('number');
+        it("the unique sprint have 40 points", function() {
+          expect(sprints[0].points).toEqual(40);
         });
       });
 
-      describe('when there is two features one with 2 points and other with no estimate', () => {
+      describe('with 3 stories of 4 points and unstarted', () => {
         const stories = [
           {
             id: 1,
-            position: "3.2",
+            position: "1",
             state: "unstarted",
-            estimate: "",
+            estimate: 4,
             storyType: "feature"
           },
           {
             id: 2,
-            position: "4.2",
+            position: "2",
             state: "unstarted",
-            estimate: 2,
+            estimate: 4,
+            storyType: "feature"
+          },
+          {
+            id: 3,
+            position: "3",
+            state: "unstarted",
+            estimate: 4,
             storyType: "feature"
           },
         ];
 
-        const project = {
-          startDate: moment().format(),
-          iterationLength: 1,
-          defaultVelocity: 8
-        };
-
         const sprints = Iteration.groupBySprints(
           stories,
           project,
-          undefined,
+          initialSprintNumber,
           []
         );
 
-        it('returns 2', () => {
-          expect(sprints[0].points).toBe(2);
+        it("return an array with 5 items", () => {
+          expect(sprints.length).toEqual(5);
+        });
+
+        it("first element have 4 points", () => {
+          expect(sprints[0].points).toEqual(4);
+        });
+
+        it("second element have 0 points", () => {
+          expect(sprints[1].points).toEqual(0);
+        });
+
+        it("second element have 2 points", () => {
+          expect(sprints[1].remainingPoints).toEqual(2);
         });
 
-        it('returns a number', () => {
-          expect(typeof (sprints[0].points)).toBe('number');
+        it("second element is filler", () => {
```

Comment:
how about changing to:
```suggestion
        it("checks if the second element is filler", () => {
```

also, look for this on your previous `it` blocks, they should be written as a regular sentence, like: `it('does something')`

Reply:
Humm, right :thinking: . Thank you!